### DISCUSSION TITLE
Custom arguments and options

### DISCUSSION THREAD

**Discussion by countnazgul** [Akryum/vue-observe-visibility#40] [Q&A]
It might be a silly question but when passing additional arguments: 

`v-observe-visibility="(isVisible, entry) => visibilityChanged(isVisible, entry, customArgument)"`

how this can be used with other options (like `throttle`, `once` etc) ?

**Comment by lukepearce** [comment]
Not a silly question, I'm wondering the same thing.

**Comment by Akryum** [comment]
Put `"(isVisible, entry) => visibilityChanged(isVisible, entry, customArgument)` on the `callback` option.
